### PR TITLE
Improved Advent Performance 

### PR DESCRIPTION
**Pull request by tianyu-z**

### CODE REVIEW

File: omnigan/trainer.py [hunk 1]
```diff
@@ -622,24 +622,38 @@ def get_masker_loss(self, multi_domain_batch):  # TODO update docstrings
...
-                            prob = torch.cat([prediction, pred_prime], dim=1)
 
-                            update_loss = self.losses["G"]["tasks"][update_task][
-                                "advent"
-                            ](
-                                prob.to(self.device),
-                                self.source_label,
-                                self.D["m"]["Advent"],
+                    if batch_domain == "r":
+                        pred_prime = 1 - prediction
```
Comment (melisandeteng) [inline omnigan/trainer.py:627]: what does prime stand for ?

File: shared/trainer/defaults.yaml [hunk 2]
```diff
@@ -124,6 +125,7 @@ dis:
   m:
     <<: *default-dis
     multi_level: false
+    architecture: OmniDiscriminator # can be [default | OmniDiscriminator]
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:128]: can we change default to base or something else ?

File: shared/trainer/defaults.yaml [hunk 3]
```diff
@@ -169,10 +171,13 @@ train:
       classifier: 1
     C: 1
     advent:
-      seg_main: 1
-      seg_aux: 0
-      adv_main: 1
-      adv_aux: 0
+      ent_main: 0.5 # the coeffcient of the MinEnt loss that directly minimize the entropy of the image
+      ent_aux: 0.0 # the coresponding coeffcient of the MinEnt loss of second output
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:175]: typo corresponding coefficient

File: omnigan/trainer.py (hunk 1 above)
Comment (tianyu-z) [reply omnigan/trainer.py:627]: I changed its name with a better one

File: shared/trainer/defaults.yaml (hunk 2 above)
Comment (tianyu-z) [reply shared/trainer/defaults.yaml:128]: Sure!

**Pull request closed without merging**